applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -298,22 +326,33 @@ static int handle_at_sleep(const char *at_cmd, enum shutdown_modes *mode)
 		}
 		if (shutdown_mode == SHUTDOWN_MODE_IDLE) {
 			slm_at_host_uninit();
-			enter_idle();
+			enter_idle(true);
 			*mode = SHUTDOWN_MODE_IDLE;
 			ret = 0; /*Will send no "OK"*/
 		} else if (shutdown_mode == SHUTDOWN_MODE_SLEEP) {
 			slm_at_host_uninit();
-			enter_sleep(true);
+			enter_sleep(false);
 			ret = 0; /* Cannot reach here */
```

Comment:
This comment seems to be outdated. Even `enter_sleep(true)` can return when `CONFIG_SLM_GPIO_WAKEUP` is not enabled.

Reply:
Yes, updated